packages/core/admin/admin/src/StrapiApp.js

```diff
@@ -227,7 +227,19 @@ class StrapiApp {
     }
 
     if (this.customConfigurations?.theme) {
-      merge(this.configurations.themes.light, this.customConfigurations.theme);
+      const darkTheme = this.customConfigurations.theme.dark;
+      const lightTheme = this.customConfigurations.theme.light;
+
+      if (!darkTheme && !lightTheme) {
+        console.warn(
```

Comment:
how many times is this going to fire in the console? Will it pollute user's console? Should it be wrapped in a `logOnce` HOC?

cc @ronronscelestes

Reply:
from my tests it only happens when we first build the app (e.g. it would happen if you refresh the application, open the application in a new tab)
I don't think once would be enough tbh and I don't know if there would be a middle ground for that, do you have something in mind @joshuaellis 
wdyt @gu-stav?

Reply:
I think if it only fires once that's the ideal situation ⭐

Reply:
perfect thanks 🙏